app/services/notices/setup/submit-contact-type.service.js

```diff
@@ -39,7 +39,8 @@ async function go(sessionId, payload, yar) {
 
   const _submittedData = {
```

Comment:
I know it is not in this PR. But just spotted that this var name needs to be corrected.

```suggestion
  const submittedData = {
```